libs/designsystem/src/lib/components/icon/icon.component.ts

```diff
@@ -34,7 +34,7 @@ export class IconComponent implements OnChanges {
   set icon(icon: Icon) {
     // If icon are not found, set default icon
     if (!icon && (this.name || this.customName)) {
-      console.warn(`Icon with name "${this.name || this.customName}" was not found.`);
+      console.warn(this.showWarning());
```

Comment:
Consider either renaming method for getting warning message or move `console.warn` to warning method.

Reply:
Renamed method